apps/desktop/src/lib/stacks/stackService.svelte.ts

```diff
@@ -97,6 +116,29 @@ function injectEndpoints(api: ClientState['backendApi']) {
 					params: { projectId, ...commitData }
 				}),
 				invalidatesTags: [ReduxTag.StackBranches, ReduxTag.Commit]
+			}),
+			/**
+			 * Does not support merge commits, i.e. 2 parent oldCommitId's yet.
+			 */
+			getCommitChanges: build.query<
+				TreeChange[],
+				{ projectId: string; oldCommitId: string; newCommitId: string }
+			>({
+				query: ({ projectId, oldCommitId, newCommitId }) => ({
+					command: 'commit_changes',
+					params: { projectId, oldCommitId, newCommitId }
+				}),
+				providesTags: [ReduxTag.CommitChanges]
+			}),
```

Comment:
I just spoke with @Byron, he's going to change the api so we don't have to pass in the old commit id, and change the endpoint name in the same go. We'll have something more like `get_commit_changes`, taking only a `commit_id`.

Reply:
See #7346!

Reply:
Awesome, I see he's merged that. I've updated this as well :+1: